connectivity/check/check.go

```diff
@@ -35,6 +35,9 @@ type Parameters struct {
 	Debug                 bool
 	PauseOnFail           bool
 	SkipIPCacheCheck      bool
+	Perf                  bool
+	Duration              int
+	Samples               int
```

Comment:
Change the type to `time.Duration` and rename to include the`Perf...` prefix (ref. my comment on the CLI flags definition):

```suggestion
	PerfDuration          time.Duration
	PerfSamples           int
```